src/main/scala/it/pagopa/interop/catalogmanagement/api/impl/EServiceApiServiceImpl.scala

```diff
@@ -612,12 +611,15 @@ class EServiceApiServiceImpl(
         ),
         createdAt = offsetDateTimeSupplier.get()
       )
+      updatedItem            = eService.copy(riskAnalysis =
```

Comment:
I would move this update in the behaviour.
This function is subjected to concurrent calls and we risk the situation we were talking about in the call.
While in the behaviour we are sure they are handled sequentially.

So the command would contain as before
```
  eServiceId: String,
  catalogRiskAnalysis: CatalogRiskAnalysis
```